graph/src/components/store.rs

```diff
@@ -86,8 +86,8 @@ pub struct EntityQuery {
     /// ID of the subgraph.
     pub subgraph_id: SubgraphDeploymentId,
 
-    /// The name of the entity type.
-    pub entity_type: String,
+    /// The name of the entity types being queried.
```

Comment:
The comment should point out that the intended semantics are a `UNION ALL` of these entity types (rather than a join across them or some such)

Reply:
I've improved the comment.